packages/start-frontend/__tests__/cli.test.ts

```diff
@@ -1,168 +1,190 @@
 import { describe, test, expect, beforeAll, afterAll } from "vitest";
 import path from "node:path";
-import fse from "fs-extra";
-import child, { ChildProcessWithoutNullStreams } from "node:child_process";
 import util from "node:util";
-import concat from "concat-stream";
-import { spawn } from "node:child_process";
+import { execFile, exec, execSync } from "node:child_process";
+import fse from "fs-extra";
```

Comment:
Minor suggestion, but what do you think about sorting the imports? (`Opt + Shift + O` on Mac VS Code)

It would be nice if there's a way to include it in the pre-commit script.